target_info/target_info.lua

```diff
@@ -0,0 +1,82 @@
+local math = require('math')
+local string = require('string')
+local ui = require('ui')
+local target = require('target')
+
+local text_color = ui.color.lightgrey
+local config_state = {
+    title = 'Target Info',
+    style = 'chromeless',
+    x = 100,
+    y = 0,
+    width = 130,
+    height = 48,
+    color = ui.color.black,
+}
+function math.round(num, prec)
+    local mult = 10^(prec or 0)
+    return math.floor((num * mult + 0.5) / mult)
+end
+
+ui.display(function()
+    ui.window('Target_Info', config_state, function()
+        local entity = target.st or target.t
+        if entity then
+           
+            local entity_speed = ((entity.status == 5 or entity.status == 85) and math.round((100 * (entity.movement_speed / 4)),2)) or math.round((100 * (entity.movement_speed / 5 - 1)),2)
+            local entity_hex = (entity.id % 0x1000)
```

Comment:
This will not work on player characters. If you want what I think you want you want `entity.index`. That *will* work on player characters as well.

Reply:
Are you sure because if so then targetinfo on win4 has been wrong a really long time.

Reply:
Ok, double checked win4 version and it's the same. Are you sure about this?

Reply:
Index is definitely what you want for placeholder info and whatnot,

Reply:
Alright, I see totally works for NPCs but not PCs like stated. I'll fix v4 version too.